next.config.js

```diff
@@ -18,10 +18,11 @@ module.exports = {
   webpack(config, options) {
     config.experiments = config.experiments || {};
     config.experiments.topLevelAwait = true;
-    // config.optimization = {
-    //   providedExports: false,
-    // };
-    // options.nextRuntime = 'edge';
+    if (options.isServer) {
+      config.resolve.fallback = {
+        '@ipfs-shipyard/ignite-metrics': false,
```

Comment:
one of the recommended methods for fixing the backend importing things it's not supposed to.

Reply:
is this due to same reason comment on the types above? nextjs importing cjs package?

Reply:
marking it as external doesn't solve the problem unfortunately, but also, neither does setting the fallback to false. I'm removing these changes as they're unnecessary

Reply:
@djmcquillan it was one of the ways recommended to tell next.js not to import something on the backend, and only allow importing of it on the frontend. But it doesn't seem to work, the noSSR import method seems to be the way.